polysemy-RandomFu/polysemy-RandomFu.cabal

```diff
@@ -63,11 +63,13 @@ test-suite polysemy-RandomFu-test
   build-depends:
       base >=4.7 && <5
     , hspec >=2.0
+    , mwc-random
```

Comment:
Please put bounds in here, even though it's only for the test.

Reply:
Sorry!  I mean in the package.yaml.

Reply:
Turns out `mwc-random` isn't even needed.

Should I still put a bound for `vector` though? I'd probably use `vector >= 0.7` since that's the bound used by `mwc-random`.